### DISCUSSION TITLE
How to print an empty list?

### DISCUSSION THREAD

**Discussion by GuilhermeCamposo** [schibsted/jslt#270] [Q&A] [answered]
I'm using JSLT 0.1.11 with Apache Camel and this is my spec: 

```
{
  "account" : $exchange.properties.account,
  "lineItems": $exchange.properties.line-items,
  "opportunity": $exchange.properties.trigger
}
```

`lineItems`, for example, can be an empty list but when that happens the field is omitted from the result. Am I missing something?

**Comment by GuilhermeCamposo** [comment]
https://github.com/schibsted/jslt/blob/master/core/src/main/java/com/schibsted/spt/data/jslt/impl/ObjectExpression.java#L90

I can see here that empty lists([]) are filtered

**Comment by larsga** [comment, marked as answer]
When you set up JSLT processing you can change the filter that's used to get exactly the behaviour that you want. In Apache Camel this is the `objectFilter` config option, although I'm not sure exactly how you can set it.

**Reply by GuilhermeCamposo** [reply]
Thanks. I will look for those instructions in the Apache Camel community.
